javaslang/src/main/java/javaslang/collection/Multimap.java

```diff
@@ -297,8 +297,7 @@ default int length() {
     @Override
     default <U> Seq<U> map(Function<? super Tuple2<K, V>, ? extends U> mapper) {
         Objects.requireNonNull(mapper, "mapper is null");
-        // don't remove cast, doesn't compile in Eclipse without it
-        return (Seq<U>) iterator().map(mapper).toStream();
+        return Stream.ofAll(iterator()).map(mapper);
```

Comment:
Nope :) The original expression `return (Seq<U>) iterator().map(mapper).toStream();` saves `n` Stream.Cons instances compared to the suggested solution.